mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -452,29 +452,62 @@ LogicalResult PopulateParamsAccel::obtainTuningParameters(
 const InitParamsAccel
 PopulateParamsXDL::initParameters[PopulateParamsXDL::nInitParameters] = {
   // M/block N/block K/block M/wave N/wave kPack forceUnroll bCopyMore
+  {128, 256, 4, 64, 128, 4, true, true}, 
+  {128, 128, 8, 64, 64, 1, true, true},  
   {128, 128, 4, 64, 64, 4, true, true},
-  {64, 64, 8, 32, 32, 4, true, true},
-  {32, 64, 4, 32, 64, 4, true, true},
-  {32, 64, 2, 8, 64, 4, true, true},
-  {4, 64, 16, 4, 64, 1, true, true}
+  {128, 128, 2, 64, 64, 8, true, true}, 
+  {64, 256, 8, 64, 64, 1, true, true},   
+  {64, 128, 2, 32, 64, 4, true, true},   
+  {64, 64, 4, 32, 32, 4, true, true},    
+  {64, 64, 4, 32, 32, 8, true, true},   
+  {64, 64, 4, 16, 64, 4, true, true},   
+  {64, 64, 2, 32, 32, 8, true, true},   
+  {32, 64, 4, 32, 64, 4, true, true},   
+  {32, 64, 4, 8, 64, 4, true, true},   
+  {32, 64, 2, 32, 32, 8, true, true},   
+  {32, 32, 8, 16, 16, 4, true, true}, 
+  {32, 64, 2, 8, 64, 4, true, true},   
+  {16, 64, 8, 16, 16, 4, true, true}, 
+  {16, 32, 8, 16, 16, 8, true, true},  
+  {16, 16, 8, 16, 16, 8, true, true}, 
+  {8, 64, 8, 4, 64, 8, true, true}   
```

Comment:
Quick checks:
1. Were there no "4,*" configs near the top of tuning? I could've sworn I've seen them before.
2. Since @pfultz2 raised the importance of this point - are the best (or close to best) configs for the gemms in the tier 1 models all on this new list?
3. Does at least one of these configs apply to all problem sizes?

(for 3, you might want to chase through the applicability checks in the tuning code to make sure there's one that'll always work)

Reply:
Hi @krzysz00 , about point 3) I was under the impression that every config is applicable to any size, given the right amount of padding. 
About point 2., instead white-listing configs, can we simply add checks that black list impossible configurations (e.g., blocksize ==2048)? It will be the tuner's responsibility to decide, then, which one of the "possible" configurations should be considered